src/Angular.js

```diff
@@ -1532,7 +1532,13 @@ function getNgAttribute(element, ngAttr) {
 
 function allowAutoBootstrap(document) {
   var script = document.currentScript;
-  var src = script && script.getAttribute('src');
+
+  // If the `currentScript` property has been clobbered just return false, since this indicates a probable attack
+  if (!(script instanceof window.HTMLScriptElement || script instanceof window.SVGScriptElement)) {
```

Comment:
Hmm, OK. I am concerned that one could clobber the `currentScript` to become null?